distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -396,6 +396,10 @@ def __init__(self, *args, n=1, name=None, nthreads=None, **kwargs):
     def status(self):
         return self.workers[0].status
 
+    @status.setter
+    def status(self, value):
+        raise NotImplementedError()
+
```

Comment:
Was curious about this as well -- it's because this inherits from `Worker`, where `status` isn't read-only. Without this it complains that we are swapping a read-write property with a read-only one, which, fair enough.

Reply:
Reinstating plugins=